docs/visual-basic/misc/bc33017.md

```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: Conversion operators must be declared either 'Widening' or 'Narrowing"
```

Comment:
There are still cases where there is a missing quote, like this one.

Reply:
@Youssef1313 Yes, sorry. I was waiting for a possible fix on the scanning tool and haven't updated these yet. I hope to have news today.

Reply:
fixed them manually after all